std/php7/Boot.hx

```diff
@@ -61,6 +61,13 @@ class Boot {
 	}
 
 	/**
+		Builds Haxe Array from array declaration syntax.
+	**/
+	public static function array( arr:NativeIndexedArray<Dynamic> ) : Array<Dynamic> {
```

Comment:
Could this be changed to:
`public static function array<T>(arr:NativeIndexedArray<T> ) : Array<T>` ?
Seems more correct to me, but won't matter much, I guess.